### ISSUE TITLE
Improvisation in What's In It section 

### ISSUE THREAD

**Issue by Hemant5104** [ritika728/Wellness_Wave#55]
Adding links to different divisions of what's in it section and adding a new part "Diet" in it .

**Comment by Zenitssuu** [comment]
> Adding links to different divisions of what's in it section and adding a new part "Diet" in it .

which links should we add

**Comment by ritika728** [comment]
@Hemant5104 since you are already assigned to an issue #47 , I won't be able to assign this to you as of now

**Comment by ritika728** [comment]
@Hemant5104 do you wish to create a new part diet in the wellness section ?

**Comment by Hemant5104** [comment]
Yes, I will provide content regarding required supplement and food

**Comment by Zenitssuu** [comment]
i want to work in this issue under JWOC

**Comment by Hemant5104** [comment]
I will link fitness training section with 30 days workout challenge
Create a diet section with suitable content beside it
Link wellness to wellness in navbar
Community support with Testimonials and Faqs
And I haven't decided what to do with the other two
I'll welcome any suggestions coming from the fellow contributors.

**Comment by Hemant5104** [comment]
@ritika728 Please add this under JWOC'24 and also add the difficulty tag

**Comment by ritika728** [comment]
@Hemant5104 go ahead
Make sure to keep the UI of the diet section in accordance with other pages.

**Comment by Hemant5104** [comment]
Mam ,Can you tell me what link would you like to add in settings and contact section

**Comment by ritika728** [comment]
> Mam ,Can you tell me what link would you like to add in settings and contact section

Remove the settings section, link contact with the contact section on the home page
